src/status_im2/contexts/onboarding/create_profile/view.cljs

```diff
@@ -1,12 +1,18 @@
 (ns status-im2.contexts.onboarding.create-profile.view
-  (:require [quo2.core :as quo]
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
             [quo2.foundations.colors :as colors]
-            [react-native.core :as rn]
             [status-im2.contexts.onboarding.create-profile.style :as style]
-            [utils.i18n :as i18n]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
             [status-im2.contexts.onboarding.common.background.view :as background]
+            [status-im2.contexts.onboarding.select-photo.method-menu.view :as method-menu]
             [utils.re-frame :as rf]))
 
+(def default-color :blue)
```

Comment:
At most places we have :default color as :primary, but it looks like figma colors file don't even have primary color anymore 
We probably need to change this too on all the places. (probably in separate PR)

Reply:
probably this should just be in constants or something like that.